web/client/utils/MapInfoUtils.js

```diff
@@ -17,7 +17,7 @@ const MapInfoUtils = {
      * specifies which info formats are currently supported
      */
     //           default format ↴
-    AVAILABLE_FORMAT: ['TEXT', 'PROPERTIES', 'HTML', 'TEMPLATE'],
+    AVAILABLE_FORMAT: ['HIDDEN', 'TEXT', 'PROPERTIES', 'HTML', 'TEMPLATE'],
```

Comment:
I don't think this is not needed

Reply:
<img width="831" alt="Screenshot 2020-11-04 at 13 14 31" src="https://user-images.githubusercontent.com/30922279/98098655-af066f80-1e9f-11eb-9fcf-a3a7b799d238.png">

DefaultSettings calls the function `MapInfoUtils.getAvailableInfoFormat()` which determines what settings are available in TOC, is there another way to go about it?

Reply:
`HIDDEN` is not a real featureInfo  format, so I think you should handle the accordion to set/unset properly the `queryable` flag, and keeping the feature info format unchanged, as well as the utilities dedicated to it.
You can append it here, as a fake info format, for the accordion. Then when click, check it and set/unset queryable accordingly, or set the current info format.

Reply:
So what I have done is append `HIDDEN` to the result of `getAvailableInfoFormat()`. I also Append it to the MapInfoUtils.INFO_FORMATS as that is used throughout the file